client/stpIfPol_service.go

```diff
@@ -0,0 +1,53 @@
+package client
+
+import (
+	"fmt"
+
+	"github.com/ciscoecosystem/aci-go-client/models"
+)
+
+func (sm *ServiceManager) CreateSpanningTreeInterfacePolicy(name string, description string, stpIfPolattr models.SpanningTreeInterfacePolicyAttributes) (*models.SpanningTreeInterfacePolicy, error) {
+	rn := fmt.Sprintf("infra/ifPol-%s", name)
+	parentDn := fmt.Sprintf("uni")
+	stpIfPol := models.NewSpanningTreeInterfacePolicy(rn, parentDn, description, stpIfPolattr)
+	err := sm.Save(stpIfPol)
+	return stpIfPol, err
+}
+
+func (sm *ServiceManager) ReadSpanningTreeInterfacePolicy(name string) (*models.SpanningTreeInterfacePolicy, error) {
+	dn := fmt.Sprintf("uni/infra/ifPol-%s", name)
```

Comment:
Can we move "uni/infra/ifPol-%s" as a constant int he model file?

Reply:
ok